libs/frontend/feature-preview/src/lib/preview-component.ts

```diff
@@ -31,11 +31,11 @@ export const PreviewComponent = () => {
     setup(props: ILiveComponentProps) {
       const { site, component } = toRefs(props)
       const { custom } = computeEvents(site.value, component.value)
-      registerCustomEvents(component.value, custom, null, true)
+      registerCustomEvents(component.value, custom, null, false)
 
       onMounted(() => {
         const { custom } = computeEvents(site.value, component.value)
-        registerCustomEvents(component.value, custom, null, false)
+        registerCustomEvents(component.value, custom, null, true)
```

Comment:
I noticed that if we use `document.getElementById(componentId)` in the appear event of a component, it always return `null`. I think the `true` and `false` here should be swapped.

Reply:
Makes sense to me, but I wonder if it's a mistake or there was some other reason 🤔 
I'll keep an eye out!